src/cls/statelog/cls_statelog.cc

```diff
@@ -279,19 +279,10 @@ static int cls_statelog_check_state(cls_method_context_t hctx, bufferlist *in, b
     return -EINVAL;
   }
 
-  string obj_index;
```

Comment:
@weiqiaomiao it certainly looks clear that even if cls_cxx_map_get_val were being used for a side effect, the same logic is executed in get_existing_entry()